rust/src/lib.rs

```diff
@@ -486,8 +490,8 @@ impl Ceed {
     /// # let ceed = libceed::Ceed::default_init();
     /// let qf = ceed.q_function_interior_by_name("Mass1DBuild".to_string());
     /// ```
-    pub fn q_function_interior_by_name(&self, name: String) -> crate::qfunction::QFunctionByName {
-        crate::qfunction::QFunctionByName::create(self, name)
+    pub fn q_function_interior_by_name(&self, name: impl Into<String>) -> QFunctionByName {
```

Comment:
Can we just use `&str`?

Reply:
Do you mean everywhere?

Reply:
We don't need a `String` unless we want ownership inside the function. I think `&str` is preferred unless it's specifically un-ergonomic.